### PR TITLE
Course sosp

### PR DESCRIPTION
**Pull request by bbbert**
### Pull Request Overview

This pull request updates the text describing the BLE ESS service, and renames excercises to exercises.

### Testing Strategy

This pull request contains only documentation changes.

### Documentation Updated

- [ ] Kernel: The relevant files in `/docs` have been updated or no updates are required.
- [ ] Userland: The application README has been added, updated, or no updates are required.

### Formatting

- [ ] `make formatall` has been run.

### CODE REVIEW

File: doc/courses/sosp/application.md [hunk 1]
```diff
@@ -192,10 +192,13 @@ we can use Tock to provide the BLE
 [Environmental Sensing Service](https://www.bluetooth.com/specifications/assigned-numbers/environmental-sensing-service-characteristics)
 (ESS).
 
-Currently, the Tock libraries do not support directly interacting with the BLE
-radio. However, we can still access BLE by loading an additional process on the
-board as a service and sending it commands over Tock's inter-process
-communication (IPC) mechanism.
+Tock implements raw BLE functionality in kernelspace and exposes this to
```
Comment (brghena) [inline doc/courses/sosp/application.md]: I'd be a little careful with this sentence. We don't really implement any BLE functionality in the kernel for SAM4L-based boards. Instead we run an application using Nordic libraries and it ships everything to the nRF over UART.

Comment (bbbert) [reply doc/courses/sosp/application.md]: Good point. Should it say "exposes BLE functionality from kernelspace" instead, leaving the question of who actually implemented BLE unanswered?

**Comment by brghena** [comment]
Looks good to me. I'll leave this for @alevy to merge though, since I don't want to screw up anything he's editing.

**Comment by brghena** [comment]
@bbbert Would you mind rebasing this PR? Github is angry with it

**Pull request merged**